mix.exs

```diff
@@ -27,6 +37,7 @@ defmodule Hummingbird.MixProject do
       {:elixir_uuid, "~> 1.2"},
       {:opencensus_honeycomb, "~> 0.2.1"},
       {:credo, "~> 0.8", only: [:dev, :test], runtime: false},
+      {:dialyxir, "~> 1.0.0-rc.4", only: [:dev, :test], runtime: false},
```

Comment:
oh god help us

Reply:
@the-mikedavis only because types are really helpful for docs and libs.  still love dialyzer... just not for our apps.